crates/katana/storage/db/src/tables/v0.rs

```diff
@@ -38,6 +38,11 @@ define_schema_enum! {
     (StorageChangeSet, TableType::DupSort)
 ]}
 
+// TODO(kariy): maybe add database changelog ?
```

Comment:
Agree on that, we may have this changelog in a `txt`/`md` file into the tables, or are  you looking to have this info directly into the code comments?